pkg/drivers/kic/kic.go

```diff
@@ -202,11 +202,13 @@ func (d *Driver) GetSSHKeyPath() string {
 
 // GetURL returns ip of the container running kic control-panel
 func (d *Driver) GetURL() (string, error) {
-	p, err := oci.HostPortBinding(d.NodeConfig.OCIBinary, d.MachineName, d.NodeConfig.APIServerPort)
-	url := fmt.Sprintf("https://%s", net.JoinHostPort("127.0.0.1", fmt.Sprint(p)))
+	ip, err := d.GetIP()
 	if err != nil {
-		return url, errors.Wrap(err, "api host port binding")
+		return "", err
 	}
+	// because of libmachine problem with detecting docker port https://github.com/kubernetes/minikube/issues/7017
+	// we have to do this hack https://github.com/docker/machine/blob/master/libmachine/provision/utils.go#L159_L175
```

Comment:
It's unclear what the hack actually is - it looks fairly normal to me. A more useful comment would be:

`// oci.HostPortBinding can be wrong, so hardcode the port because of ...`

I like the links to the issue, but don't understand the link to the code. Also, the code link will go stale at some point, since it doesn't pin to a revision.